Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ Due to package verification, the latest version below is the unpublished version
 however, it has to be formatted properly to pass verification tests.
 
 ## [Unreleased] - yyyy-mm-dd
+
+### Fixed
+- Avoid potential crashes from NullReferenceException in FireStateChangeNotifications [ISXB-724](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-724)
```

Comment:
This URL doesn't work. It seems the ticket is internal and not available via public tracker? In that case remove the hyperlink.

Reply:
Thanks, fixed in 1c1c67b459aff7a28a55edf33f84e22648268d1b and I added a minor other fix too.